firefox_ui_harness/runtests.py

```diff
@@ -87,9 +44,10 @@ def startTestRunner(runner_class, options, tests):
 
 
 def cli(runner_class=FirefoxUITestRunner, parser_class=FirefoxUIOptions):
-    parser = parser_class(usage='%prog [options] test_file_or_dir <test_file_or_dir> ...')
+    parser = parser_class(usage='%prog [options] manifest <test_file_or_dir> ...')
```

Comment:
The arguments can be a manifest, a test file, or a directory. So we have three options. Maybe I should simple leave this change out of this PR and file a separate bug for it.